Resources/Prototypes/Entities/Constructible/Ground/wires.yml

```diff
@@ -95,8 +95,13 @@
   parent: ApcExtensionCable
   components:
   - type: NodeContainer
-    nodes:
-    - !type:AdjacentNode
-      nodeGroupID: HVPower
-    - !type:AdjacentNode
-      nodeGroupID: Apc
+    nodeTypes: { HVPower : ["AdjacentNode"], Apc : ["AdjacentNode"] }
```

Comment:
Is this supposed to change? It seems like a bad merge conflict from 01b10cb6876cbbd3e476cf97501f4a892a3e41f5, as "nodeTypes" is no longer read anywhere in NodeContainerComponent but "nodes" is.

Reply:
Looks like this on latest master: 
`   nodes:
    - !type:AdjacentNode
      nodeGroupID: Apc`